samples/drivers/mspi/mspi_flash/boards/b_u585i_iot02a.overlay

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2025 EXALT Technologies.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Replace the octospi2 by a "st,stm32-ospi-controller" compatible */
+/delete-node/ &octospi2;
+
+/ {
+	aliases {
+		/* The sample gets the DT_ALIAS(flash0) as the mspi device */
+		flash0 = &mx25lm51245;
+	};
+
+	soc {
+		octospi2: spi@420d2400 {
+			compatible = "st,stm32-ospi-controller";
+			reg = <0x420d2400 0x400>, <0x70000000 DT_SIZE_M(256)>;
+			interrupts = <120 0>;
+			clock-names = "ospix", "ospi-ker", "ospi-mgr";
+			clocks = <&rcc STM32_CLOCK(AHB2_2, 8)>,
+				 <&rcc STM32_SRC_SYSCLK OCTOSPI_SEL(0)>,
+				 <&rcc STM32_CLOCK(AHB2, 21)>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			op-mode = "MSPI_CONTROLLER";
+			status = "okay";
```

Comment:
Prefer have the `status` property last among the node properties. Not a blocking issue.